interactive-bots/helloworld/nicku1/notes.md

```diff
@@ -0,0 +1,5 @@
+First problem: Installing bots development environment. For no reason after cloning rep, command ./tools/provision doesn't work. I re-clone it in after entering to vargant, works fine now :)
+
+Second problem: Created zuliprc-my-bot folder outside the rep, so I cannot launch the bot. Moving it into "zulip" folder solved everything ;)

```

Comment:
You can specify a full path or a relative path. Example, if the folder is outside the repo, you could run it like `zulip-run-bot -c ../../somewhere/zuliprc`.